src/components/Alphabet/Letter/Letter.js

```diff
@@ -1,34 +1,62 @@
 import React from 'react';
 import styles from './Letter.css';
+import classnames from 'classnames';
 
-const checkLetter = (letter, id, onLetterClick) => {
-    const url = `/phrases/check`;
-    return fetch(url, {
-        headers: {
-            'Content-Type': 'application/json',
-            'Accept': 'application/json',
-        },
-        method: 'post',
-        body: JSON.stringify({
-            id,
-            letter,
+class Letter extends React.PureComponent {
+    state = {
+        isLetterCorrect: null,
+        usedLetter: false,
+    };
+
+    checkLetter = (letter, id) => {
+        const url = `/phrases/check`;
+        return fetch(url, {
+            headers: {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+            },
```

Comment:
Do you need those 'Content-Type' and 'Accept' headers here?

Reply:
If not here, then where? Or do you suggest that headers are unnecessary at all?
When I read about post method headers were attached in most cases
[https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch](url)

Reply:
It might be that by default - try to delete that and see if it still works fine.

Reply:
Still valid?

Reply:
It looks that headers are necessary.